FileIO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -71,21 +72,25 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
 
     MeshLib::NodePartitionedMesh* mesh = nullptr;
 
+    /// Prefix of the filename of the partitioned mesh files.
+    std::string const filename_prefix =
+        boost::erase_last_copy(filename_prefix, ".vtk");
```

Comment:
@endJunction Tested with the above correction. You can amend the commit with that.